packages/stores/types.ts

```diff
@@ -384,7 +384,7 @@ type TCommonStoreError = {
     type?: string;
 };
 
-type TCommonStore = {
+export type TCommonStore = {
```

Comment:
```suggestion
type TCommonStore = {
```

Reply:
This TCommonStore type is being used in the `cfd/src/_common/base/server_time.ts` file

Reply:
@arshad-rao-deriv Please don't export types from this file, You can access it through `TStores['common']` if needed.